### DISCUSSION TITLE
No schematic match this project

### DISCUSSION THREAD

**Discussion by ianvink** [cyrilletuzi/vscode-angular-schematics#916] [Q&A]
I added the lic, then rebooted, then right clicked a folder and choose to Generate a file'
![image](https://github.com/cyrilletuzi/vscode-angular-schematics/assets/112916/4bb7cc53-5fd5-42cb-b409-db59c3d73bd9)

**Comment by ianvink** [comment]
I tried this as well, "[use the configuration helper to force the dependencies detected](command:angular-schematics.configuration)."

**Comment by cyrilletuzi** [comment]
Hi @ianvink,

Thank you for your subscription to the Pro edition. I have sent you an email to help you to get things right, it will be easier and faster.
